tests/test_markdown.py

```diff
@@ -2,36 +2,44 @@
 import glob
 import os
 
+import cmarkgfm
 import pytest
 
 from readme_renderer.markdown import render, variants
 
 
+MD_FIXTURES = [
+    (fn, os.path.splitext(fn)[0] + ".html", variant)
+    for variant in variants
+    for fn in glob.glob(
+        os.path.join(
+            os.path.dirname(__file__),
+            "fixtures",
+            "test_" + variant + "*.md"
+        )
+    )
+]
+
+
 @pytest.mark.parametrize(
     ("md_filename", "html_filename", "variant"),
-    [
-        (fn, os.path.splitext(fn)[0] + ".html", variant)
-        for variant in variants
-        for fn in glob.glob(
-            os.path.join(
-                os.path.dirname(__file__),
-                "fixtures",
-                "test_" + variant + "*.md"
-            )
-        )
-    ],
+    MD_FIXTURES,
 )
 def test_md_fixtures(md_filename, html_filename, variant):
     # Get our Markup
     with io.open(md_filename, encoding='utf-8') as f:
-        md_markup = f.read()
+        md_markup = f.read().strip()
 
     # Get our expected
     with io.open(html_filename, encoding="utf-8") as f:
-        expected = f.read()
+        expected = f.read().strip()
 
-    assert render(md_markup, variant=variant) == expected
+    assert render(md_markup, variant=variant).strip() == expected
```

Comment:
Why is `.strip()` necessary here? Can we just add/remove newlines from the fixtures as necessary?

Reply:
Sure. The reason I added this is because `bleach` can occasionally consume the trailing newline from things- especially when it fixes broken HTML. This is mostly just affecting on particular fixture, so I can likely just do that. WDYT?

Reply:
done in 93589c7